modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,11 @@
+* <<xml_business_handbook.adoc#, Business Handbook>>
+* Technical Handbook
```

Comment:
when the navigation was added in November we realized that there was a problem with the with of the columns in the text of the TechHandbook.
I see now has been corrected now. 
However all xml examples seem now to be cut on the right side. Is that a consequence of resizing the columns in the tables?

Reply:
Due to the length of the xml code I'm afrait it cannot be helped. Nonetheless, each xml snippet has a scroll bar at the bottom to view each line in full.
Maybe we could split xml snippets in smaller blocs so navigation is easier.
![Captura](https://user-images.githubusercontent.com/67598083/154459541-ebc4a713-f88d-47f6-9f02-d8b56b53a093.JPG)